knora/README.md

```diff
@@ -1,563 +1,682 @@
+    - [get_existing_projects()](#get_existing_projects)
+    - [get_project()](#get_project)
+    - [project_exists()](#project_exists)
+    - [create_project()](#create_project)
+    - [update_project()](#update_project)
+    - [get_users()](#get_users)
+    - [create_user()](#create_user)
+    - [add_user_to_project()](#add_user_to_project)
 - [Ontology methods](#ontology-methods)
-  - [get_existing_ontologies()](#get_existing_ontologies)
-  - [get_project_ontologies()](#get_project_ontologies)
-  - [ontology_exists()](#ontology_exists)
-  - [get_ontology_lastmoddate()](#get_ontology_lastmoddate)
-  - [create_ontology()](#create_ontology)
-  - [delete_ontology()](#delete_ontology)
-  - [get_ontology_graph()](#get_ontology_graph)
-  - [create_res_class()](#create_res_class)
-  - [create_property()](#create_property)
-  - [create_cardinality()](#create_cardinality)
+    - [get_existing_ontologies()](#get_existing_ontologies)
+    - [get_project_ontologies()](#get_project_ontologies)
+    - [ontology_exists()](#ontology_exists)
+    - [get_ontology_lastmoddate()](#get_ontology_lastmoddate)
+    - [create_ontology()](#create_ontology)
+    - [delete_ontology()](#delete_ontology)
+    - [get_ontology_graph()](#get_ontology_graph)
+    - [create_res_class()](#create_res_class)
+    - [create_property()](#create_property)
+    - [create_cardinality()](#create_cardinality)
 - [Lists](#lists)
-  - [create_list_node()](#create_list_node)
-  - [get_lists()](#get_lists)
-  - [get_complete_list()](#get_complete_list)
+    - [create_list_node()](#create_list_node)
+    - [get_lists()](#get_lists)
+    - [get_complete_list()](#get_complete_list)
 - [Creating instances](#creating-instances)
-  - [create_resource()](#create_resource)
-    - [values parameter](#values-parameter)
-    - [Stillimage parameter](#stillimage-parameter)
+    - [create_resource()](#create_resource)
+        - [values parameter](#values-parameter)
+        - [Stillimage parameter](#stillimage-parameter)
 - [Other methods](#other-methods)
-  - [create_schema()](#create_schema)
-  - [reset_triplestore_content()](#reset_triplestore_content)
+    - [create_schema()](#create_schema)
+    - [reset_triplestore_content()](#reset_triplestore_content)
 - [SIPI](#sipi)
-  - [Sipi()](#Sipi)
-  - [upload_image()](#upload_image)
+    - [Sipi()](#Sipi)
+    - [upload_image()](#upload_image)
 - [BulkImport](#bulkImport)
-  - [Bulkimport()](#bulkimport)
-  - [add_resource()](#add_resource)
-  - [upload()](#upload)
-  
+    - [Bulkimport()](#bulkimport)
+    - [add_resource()](#add_resource)
+    - [upload()](#upload)
+
 ## Knora
-Knora is the base class which handles all the communication with the Knora backend. In order
-to use Knora, the user has to submit his/her credentials to the Knora backend
+
+Knora is the base class which handles all the communication with the Knora backend. In order to use Knora, the user has to submit
+his/her credentials to the Knora backend
```

Comment:
```suggestion
Knora is the base class which handles all the communication with the DSP API. In order to use Knora, the user has to submit
his/her credentials to the DSP API
```
Not that it matters, but we were replacing all Knora, right?

Reply:
also `his/her` is very gender binary... one could even put `their` to be a bit more modern

Reply:
I didn't go through all the file but now that you mention it, I replaced the Knora reference with DSP or DSP API everywhere in this file. I corrected the his/her, too. Thanks for mentioning that.